tide/wall/ImageSource.cpp

```diff
@@ -77,13 +77,10 @@ Indices ImageSource::computeVisibleSet( const QRectF& visibleTilesArea,
 {
     Q_UNUSED( lod );
 
-    Indices visibleSet;
-
     if( visibleTilesArea.isEmpty( ))
-        return visibleSet;
+        return Indices{};
```

Comment:
for the record, clang does not allow "return {}"
error: chosen constructor is explicit in copy-initialization